### PR TITLE
Populate seed_data with new models' fields

### PR DESCRIPTION
**Pull request by emarc99**
Closes: #274

### CODE REVIEW

File: web_app/db/seed_data.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def create_positions(session: SessionLocal, users: list[User]) -> None:
     """
     positions = []
     for user in users:
-        for _ in range(2):
+        for _ in range(10):
```
Comment (djeck1432) [inline web_app/db/seed_data.py:48]: why did you change my code?

File: web_app/db/seed_data.py [hunk 2]
```diff
@@ -78,7 +82,7 @@ def create_airdrops(session: SessionLocal, users: list[User]) -> None:
     """
     airdrops = []
     for user in users:
-        for _ in range(2):
+        for _ in range(10):
```
Comment (djeck1432) [inline web_app/db/seed_data.py:85]: why did you change my code?

File: web_app/db/seed_data.py [hunk 3]
```diff
@@ -101,14 +106,15 @@ def create_telegram_users(session: SessionLocal, users: list[User]) -> None:
     """
     telegram_users = []
     for user in users:
-        for _ in range(2):
+        for _ in range(10):
```
Comment (djeck1432) [inline web_app/db/seed_data.py:109]: why did you change my code

**Review by djeck1432 — CHANGES_REQUESTED** [review]

File: web_app/db/seed_data.py (hunk 1 above)
Comment (emarc99) [reply web_app/db/seed_data.py:48]: "Create 10 per new models"

Comment (emarc99) [reply web_app/db/seed_data.py:48]: Was thinking of using range(1) as well.

File: web_app/db/seed_data.py (hunk 2 above)
Comment (emarc99) [reply web_app/db/seed_data.py:85]: "Create 10 per new models"

File: web_app/db/seed_data.py (hunk 3 above)
Comment (emarc99) [reply web_app/db/seed_data.py:109]: "Create 10 per new models"

**Comment by djeck1432** [comment]
@emarc99 Can't manage with task
